csgc-jekyll-src/housing.md

```diff
@@ -0,0 +1,96 @@
+	- Townhomes
+	- Near bus stops
+	- Pricier but nicer
+	- Caters more towards young professionals
+- Oak Manor
+	- Townhomes
+	- Near bus stops
+	- Mix of graduate and undergraduate students
+- [Foxridge](https://www.FoxridgeLiving.com)
+	- Apartments
+	- Near bus stops
+	- Caters more towards undergrads, but many grad students also live there
+- [Hethwood (next to Foxridge)](https://www.HethwoodLiving.com)
+	- Townhomes and houses
+	- Near bus stops
+	- Caters more towards grad students and young professionals, but some undergrads also live there
+
+#### Harding
+
+- [Windsor Hills](https://www.windsorhillsapat.net)
+	- Apartments
+	- Near bus stops
+
+#### Patrick Henry
+
+- [University Terrace](https://www.)
+	- Apartments and condos
+	- It’s close to a bus stop
+- [Terrace View](https://www.terraceviewapartments.com)
+	- Apartments
+	- It’s close to a bus stop
+
+#### Tom’s Creek
+
+- [Shawnee Apartments](https://www.cmgleasing.com)
+	- Apartments
+	- It’s close to a bus stop
+- [Chasewood Downs Apartments](https://www.cmgleasing.com)
+	- Apartments
+	- It’s close to a bus stop
+
+#### CRC
+
+- [Knollwood](https://www.cmgleasing.com)
+	- Townhouses, apartments, and condos
+	- Right next to a bus stop (but know that the busses that take you here run on a very different schedule from the main campus busses)
+- Ramble Ridge (next to Knollwood)
+	- Townhouses
+	- Near bus stops (but know that the busses that take you here run on a very different schedule from the main campus busses)
+
+#### Christiansburg
+
+- [Charleston Place Townhomes (Townside Property Management)]()
+	- Townhouses
+	- Near bus stops (but know that the busses that take you here run on a very different schedule from the main campus busses)
+	- Cater more towards grad students and young professionals
+
+### [Google Directions](https://www.google.com/maps/dir/Windsor+Hills+Apartments/University+Terrace,+Blacksburg,+VA+24060/Terrace+View+Apartments/Chasewood+Downs+Apartments/Shawnee+Apartments/Smith's+Landing+Apartments/Oak+Manor,+Blacksburg,+VA/Foxridge+Collegiate+Apartment+Homes/Charleston+Ln,+Christiansburg,+VA+24073/The+Reserve+at+Knollwood/@37.1922824,-80.4953426,12z/data=!3m1!4b1!4m62!4m61!1m5!1m1!1s0x884d958fb6ccb271:0x1b46263e584a6208!2m2!1d-80.3970311!2d37.2394331!1m5!1m1!1s0x884d957b5b6e810b:0x4530cada7d65c0d!2m2!1d-80.4204391!2d37.244557!1m5!1m1!1s0x884d956428ac6aed:0x3f5d409de35b8a38!2m2!1d-80.4258046!2d37.2439206!1m5!1m1!1s0x884d95678fdc0835:0xdb6652fa53bcaf71!2m2!1d-80.4312741!2d37.2422128!1m5!1m1!1s0x884d955d3e34b6cb:0x2d51f7eb58df0fa8!2m2!1d-80.4332636!2d37.2420793!1m5!1m1!1s0x884d953a23a80731:0x67481b327babe499!2m2!1d-80.4418296!2d37.2241193!1m5!1m1!1s0x884d9531a7d95033:0x8dd46239a01f33f2!2m2!1d-80.4472739!2d37.2231839!1m5!1m1!1s0x884d952c9e50121d:0x7fe981409c85182a!2m2!1d-80.4527595!2d37.2171116!1m5!1m1!1s0x884d93bc935f9cd1:0xa6fa0fee1bc2364a!2m2!1d-80.397512!2d37.138387!1m5!1m1!1s0x884d945475ea6a7f:0xb8cc1d741d71c195!2m2!1d-80.4047298!2d37.2029514!3e0)
```

Comment:
Not sure the Google Maps link is necessary?

Reply:
The Google Maps link creates the route that is represented within the images.
Since I couldn't find the url for all of the locations, the link would help people get some of the addresses.